src/components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
the second version of this component (when you have reserved place in the therapy) will be created in the another ticket? Isn't this also a part of this ticket? -> https://zpl.io/br8nnjO